### PR TITLE
[TIMOB-25566] iOS: Fix setting "showControls" on videoplayer when URL is changed after creation

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25566

**no tests**, because it's a UI related change.

### CODE REVIEW

**Comment by jquick-axway** [comment]
**Note to everyone:**
I've already written a "showsControls" unit test in PR #9770, which is merged in. So, the unit test is already covered. 👍

File: iphone/Classes/TiMediaVideoPlayerProxy.m [hunk 1]
```diff
@@ -134,6 +134,14 @@ - (void)configurePlayer
 {
   [self addNotificationObserver];
   [self setValuesForKeysWithDictionary:loadProperties];
+
+  // TIMOB-25566: Video controls need to be handled manually as they can
+  // cause issues if the URL is set after creation
+  id showControls = [self valueForKey:@"showControls"];
```
Comment (vijaysingh-axway) [inline iphone/Classes/TiMediaVideoPlayerProxy.m]: Property name is "showsControls". Please update.

File: iphone/Classes/TiMediaVideoPlayerProxy.m [hunk 2]
```diff
@@ -469,7 +477,10 @@ - (NSNumber *)showsControls
 
 - (void)setShowsControls:(NSNumber *)value
 {
-  [movie setShowsPlaybackControls:[TiUtils boolValue:value def:YES]];
+  [self replaceValue:value forKey:@"showControls" notification:NO];
```
Comment (vijaysingh-axway) [inline iphone/Classes/TiMediaVideoPlayerProxy.m]: Same as above. Property name is wrong.

**Review by vijaysingh-axway — CHANGES_REQUESTED** [review]

**Review by vijaysingh-axway — CHANGES_REQUESTED** [review]
Still issue is there. Controls are not hiding.

**Comment by hansemannn** [comment]
@vijaysingh-axway Not sure why I worked before, but I adjusted it to store it in the `loadProperties` which is even more clean :-).

**Comment by vijaysingh-axway** [comment]
@hansemannn  Changes looks good. I guess you can add unit test for this. This is more to setting property showsControls than UI.

**Comment by hansemannn** [comment]
@vijaysingh-axway Added

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Pull request merged**